poetry.lock

```diff
@@ -1,43 +1,172 @@
+# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
+
+[[package]]
```

Comment:
Ok so this is the poetry.lock from 21cmfast. Can you explain a bit what is going on?